src/controllers/handlers/outfits-handler.ts

```diff
@@ -0,0 +1,20 @@
+import { HandlerContextWithPath, NotFoundError, TypedEntity } from '../../types'
+import { getOutfits } from '../../logic/outfits'
+import { Outfits } from '@dcl/schemas'
+
+export async function outfitsHandler(
+  context: HandlerContextWithPath<
+    'metrics' | 'content' | 'theGraph' | 'config' | 'fetch' | 'ownershipCaches',
+    '/profile/:id'
+  >
+): Promise<{ status: 200; body: TypedEntity<Outfits> }> {
```

Comment:
If this is a public endpoint the idea will be to add the doc to API specs and then use the auto generated type (sorry, I know it's a bit annoying)